spec/system/visit_user_journey_view_certificate_page_spec.rb

```diff
@@ -43,13 +44,44 @@
       click_link 'Replace certificate'
       expect(current_path).to eql before_you_start_path(sp_component.component_type, sp_component.id, sp_component.encryption_certificate_id)
     end
+
     it 'signing certificate information and navigates to next page' do
-      certificate = create(:sp_signing_certificate)
-      sp_component = certificate.component
+      sp_signing_certificate = create(:sp_signing_certificate)
+      sp_component = sp_signing_certificate.component
       visit view_certificate_path(sp_component.component_type, sp_component.id, sp_component.signing_certificates[0])
       expect(page).to have_content 'Verify Service Provider: signing certificate'
       click_link 'Add new certificate'
       expect(current_path).to eql before_you_start_path(sp_component.component_type, sp_component.id, sp_component.signing_certificates[0])
     end
   end
+
+  context 'show signing' do
+    it 'specific information when certificate is primary' do
+      first_certificate = create(:msa_signing_certificate, component: create(:msa_component, team_id: @user.team))
+      defaults = {
+        usage: CERTIFICATE_USAGE::SIGNING,
+        value: second_certificate,
+        component: first_certificate.component
+      }
+      second_cert = Certificate.create(defaults)
```

Comment:
```suggestion
      create(:msa_signing_certificate, component: first_certificate.component)
```

Might be able to get away with using the factory? Also are we using the assigned variable anywhere?

Same for the the it block below